atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/backend/DockerizedDatabaseUri.java

```diff
@@ -33,11 +33,11 @@ public static DockerizedDatabaseUri fromUriString(String uri) {
         String[] parts = uri.trim().split(DELIMITER);
         String[] addrParts = parts[1].split(":");
         return new DockerizedDatabaseUri(
-                KeyValueServiceType.valueOf(parts[0]),
+                KeyValueServiceTypeInterface.keyValueServiceTypeFor(parts[0]),
                 InetSocketAddress.createUnresolved(addrParts[0], Integer.parseInt(addrParts[1])));
     }
 
-    public KeyValueServiceType getKeyValueServiceType() {
+    public KeyValueServiceTypeInterface getKeyValueServiceType() {
```

Comment:
Also, what else is this used for? If it's only used [here](https://github.com/palantir/atlasdb/pull/1360/files#diff-3f3d5b40164a1b372e79f78242e05d7eR47), then we should simply supply a `getKeyValueServiceConfig` method.